patches/minecraft/net/minecraft/client/Minecraft.java.patch

```diff
@@ -1,14 +1,70 @@
 --- ../src-base/minecraft/net/minecraft/client/Minecraft.java
 +++ ../src-work/minecraft/net/minecraft/client/Minecraft.java
-@@ -344,7 +344,6 @@
+@@ -328,6 +328,7 @@
+     private final Tutorial field_193035_aW;
+     long field_71421_N = -1L;
+     private String field_71465_an = "root";
++    public CrashReport integratedServerCrash = null;
+ 
+     public Minecraft(GameConfiguration p_i45547_1_)
+     {
+@@ -344,7 +345,6 @@
          this.field_152355_az = (new YggdrasilAuthenticationService(this.field_110453_aa, UUID.randomUUID().toString())).createMinecraftSessionService();
          this.field_71449_j = p_i45547_1_.field_178745_a.field_178752_a;
          field_147123_G.info("Setting user: {}", (Object)this.field_71449_j.func_111285_a());
 -        field_147123_G.debug("(Session ID is {})", (Object)this.field_71449_j.func_111286_b());
          this.field_71459_aj = p_i45547_1_.field_178741_d.field_178756_a;
          this.field_71443_c = p_i45547_1_.field_178743_b.field_178764_a > 0 ? p_i45547_1_.field_178743_b.field_178764_a : 1;
          this.field_71440_d = p_i45547_1_.field_178743_b.field_178762_b > 0 ? p_i45547_1_.field_178743_b.field_178762_b : 1;
-@@ -464,10 +463,10 @@
+@@ -395,6 +395,8 @@
+                     {
+                         try
+                         {
++                        try
++                        {
+                             this.func_71411_J();
+                         }
+                         catch (OutOfMemoryError var10)
+@@ -403,6 +405,38 @@
+                             this.func_147108_a(new GuiMemoryErrorScreen());
+                             System.gc();
+                         }
++                        }
```

Comment:
You dont need this second try, you can just add a extra catch to the existing one.

Reply:
oh yeah. derp